hs_composite_resource/templates/pages/compositeresource.html

```diff
@@ -9,4 +9,5 @@
     {# resource specific js can go here #}
     <script type="text/javascript" src="{{ STATIC_URL }}js/composite-resource.js"></script>
     <script type="text/javascript" src="{{ STATIC_URL }}js/time-series-filetype.js"></script>
+    <script type="text/javascript" src="{{ STATIC_URL }}js/jsoneditor.min.js"></script>
```

Comment:
@cuahsimarko - could you check out the jsoneditor and how it's being used in this PR?

Reply:
@pkdash this is my first skim of this PR and will be revisiting more. Are you just using this as a part of your debug/testing/dev workflow? https://github.com/josdejong/jsoneditor

Reply:
I haven't looked at the endpoints in detail yet, did skim some PyTest stuff, but are you making REST calls and maybe tools like postman for dev? I saw some ajax posts, seems good so far.